### DISCUSSION TITLE
GSoC 2023 Query

### DISCUSSION THREAD

**Discussion by silvems** [pybamm-team/PyBaMM#2847] [General]
Hi @rtimms @tinosulzer @Saransh-cpp @priyanshuone6 
I want to apply for the project "Documentation" for GSoC 2023. I am currently working on my proposal. Can you give me some details regarding how to get feedback from the project mentors on the draft proposal before final submission? Should I post it here?
I tried to join the Slack workspace but it seems restricted by email. 

Also can I work on some issues in the repo to get familiar with the codebase? I know I am a little late in terms of the deadline but I still believe I'll be able to make a decent proposal before the last date...

Regards
@silvems

**Comment by Saransh-cpp** [comment]
Hi! Please feel free to email any mentor for the Slack invite. You can then go ahead and personally share the proposal for review with any mentor on Slack.

We strongly encourage making a non-trivial contribution before you submit a proposal. You can have a look at [difficulty: easy](https://github.com/pybamm-team/PyBaMM/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+easy%22) issues to start with the contributions.
